### ISSUE TITLE
[accessibility-defect-1]: Hint text color does not have enough contrast when inside a gray card.

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#71633]
### Point of contact

@jeana-adhoc

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

[On the disability exams page in 20-10206](https://staging.va.gov/records/request-personal-records-form-20-10206/disability-exam-details) there is a memorable date component with hint text inside of a gray card.

Axe reports:
`Element has insufficient color contrast of 4.06 (foreground color: #71767a, background color: #f1f1f1, font size: 12.7pt (16.96px), font weight: normal). Expected contrast ratio of 4.5:1`

<img width="450" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/2f98e502-d4db-4906-a205-a6e3abc0cdc0">

I think this will be a design system issue.  Related tickets I've found:
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2104
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1693

The other solution, could be to change the grey of the tile card, but it is currently using the [lightest gray](https://design.va.gov/foundation/utilities/background-color) possible, `#f1f1f1` 

### Reproduction steps

1. Load the [disabilities exams page](https://staging.va.gov/records/request-personal-records-form-20-10206/disability-exam-details)
2. Run axe browser extension
3. See the error for the color contrast.

### Proposed solution or next steps

After talking with @rachelshearerux like to see if the following is possible:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/ca5d2041-5e77-4d53-861f-432f65b502b4)

1. Remove all hint text on the memorable date component, even the built in "For example: January 19 2000"
2. Change the label of the component to be all of this text: `When was your exam? (For example: January 19 2000) <br /> We don't require this information, and it's OK to estimate.`

Possible challenges: Adding a line break in the string, removing the built in hint text on the component.

### References, articles, or WCAG support

1. [Contrast (Minimum): Understanding SC 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

### Type of issue

- [X] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [X] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by jeana-adhoc** [comment]
Suggested update:

After talking with @rachelshearerux like to see if the following is possible:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/ca5d2041-5e77-4d53-861f-432f65b502b4)

1. Remove all hint text on the memorable date component, even the built in "For example: January 19 2000"
2. Change the label of the component to be all of this text: `When was your exam? (For example: January 19 2000) <br /> We don't require this information, and it's OK to estimate.`

Possible challenges: Adding a line break in the string, removing the built in hint text on the component.

The suggestion here should only apply to this form.

I'll also be filing a bug with the DST so they are aware of this issue.

**Comment by jeana-adhoc** [comment]
I've confirmed the fix on staging, where we've changed the background of the card to white with a border.

This clears the axe error!

Thanks @rhasselle-oddball !
